CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ always judge technical contributions to the project based on their technical
 merits. However, as a company, our immediate priorities can and do change often
 and sometimes without prior notice. This affects the design and decision making
 process as well as the development and review velocity. Some proposals may also
-have direct impact on our viability as a company, in which case we carefully
+have a direct impact on our viability as a company, in which case we carefully
```

Comment:
I would argue that this is actually also fine. "have impact" works without an article.